app/src/utils/enums/application.ts

```diff
@@ -30,6 +30,11 @@ export enum Initiative {
   HOUSING = 'HOUSING'
 }
 
+export enum UserStatus {
```

Comment:
Misnomer. This is the status of the access request, not the user.

Reply:
makes sense